src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -215,15 +216,15 @@ private static unsafe nuint GetIndexOfFirstNonAsciiByte_Default(byte* pBuffer, n
             goto Finish;
         }
 
-        private static unsafe nuint GetIndexOfFirstNonAsciiByte_Sse2(byte* pBuffer, nuint bufferLength)
+        private static unsafe nuint GetIndexOfFirstNonAsciiByte_Sse2OrArm(byte* pBuffer, nuint bufferLength)
```

Comment:
Shouldn't the name end with `...Sse2OrArm64`? The optimizations in this method were mainly focused on `Arm64`.

Reply:
Yup. Changed it.